src/main/scala/dpla/ingestion3/mappers/utils/Extractor.scala

```diff
@@ -134,6 +147,24 @@ trait XmlExtractor extends Extractor[NodeSeq] {
   def extractChildStrings(xValue: NodeSeq): Seq[String] = xValue.flatMap { node =>
     node.child.collect{ case Text(t) => t }.map(_.trim).filterNot(_.isEmpty)
   }
+
+  /**
+   *
+   * @param node
+   * @param attribute
+   * @return
+   */
+  def getAttributeValue(node: Node, attribute: String): Option[String] = {
```

Comment:
do you need this? Doesn't `\@` work?

Reply:
No, it doe not with with name spaced attributes for which the namespace is not declared

Reply:
Should we force some well-known prefixes into the document in the Mapper so this doesn't happen?

Reply:
Ohh that is a great idea.

Reply:
Added list of predefined namespaces to extractor. It won't help in this case but it will for other providers where namespaces are used for attributes and the namespaces are declared in the document.